Jenkinsfile.integration

```diff
@@ -41,7 +41,7 @@ node('node-06') {
 				deleteDir()
 				checkout scm
 
-				sh '''#!/bin/bash
+				sh '''
 					# Cycle postgres, install deps and prepare config
 					sudo service postgresql restart
 					rsync -axl -e "ssh -oUser=jenkins" master-01:/var/lib/jenkins/lisk/node_modules/ "$WORKSPACE/node_modules/"
```

Comment:
Using the cache is a nice to have but still optional: it should never fail the whole `sh` step.